nldb/api.py

```diff
@@ -68,4 +68,4 @@ async def serve_index():
 
 
 def serve():
-    uvicorn.run("nldb.api:app", host=UVICORN_HOST, port=UVICORN_PORT, workers=2)
+    uvicorn.run("nldb.api:app", host=UVICORN_HOST, port=UVICORN_PORT, workers=1)
```

Comment:
suggestion: the number of workers be configurable

Reply:
I set this to 1 because I'm not v clear what multiple workers mean in an async environment, esp when GCR spins up new machines when needed. Happy to change though!

Reply:
Just read https://fastapi.tiangolo.com/deployment/server-workers/ and I no longer think we should make this configurable. At least not in the context this would be run